File: src/main/java/Ex07_ConditionalExpression.java
public class Ex07_ConditionalExpression {
    public static void main(String[] args) {
        long randOne = Math.round(Math.random()); // Provide us 0 or 1 number
        long randTwo = Math.round(Math.random()); // Provide us 0 or 1 number
        String toPrint = randOne == 0 ? randTwo == 0 ? "I love Kotlin" : "I love Java" : null;
        System.out.println(toPrint);
    }
}

File: src/main/kotlin/Ex07_ConditionalExpression.kt
import kotlin.math.roundToInt

fun main() {
  val randOne: Int = Math.random().roundToInt()
  val randTwo: Int = Math.random().roundToInt()

  val toPrint: String? = if (randOne == 0 && randTwo == 0) "I love Koltin"
  else if (randTwo == 0) "I love Java"
  else null

  println(toPrint)
}